packages/bpk-component-calendar/src/BpkCalendarGridTransition.js

```diff
@@ -147,7 +147,11 @@ class BpkCalendarGridTransition extends Component {
       isTransitioning && 'bpk-calendar-grid-transition__strip--transitioning',
     );
 
-    const { min, max } = getMonthRange(rest.minDate, rest.maxDate);
+    let min;
+    let max;
+    if (rest.minDate && rest.maxDate) {
+      ({ min, max } = getMonthRange(rest.minDate, rest.maxDate));
```

Comment:
This check ensures that `getMonthRange` isn't called on `null`, which now returns `Invalid Date`.